app/src/main/java/edu/uc/haflga/financialpal/dto/Category.kt

```diff
@@ -1,9 +1,10 @@
 package edu.uc.haflga.financialpal.dto
 
-//Category Class for transactions
-//Limit parameter for transaction limit will be implemented in next sprint...
-class Category(var name: String, var limit: Limit?) {
-
+/**
+ * Category Class for transactions
+ * Limit parameter for transaction limit will be implemented in next sprint...
+ */
+data class Category(var name: String, var limit: Limit?) {
```

Comment:
Changed this to a data class so that you had the built in functionality that it should have.

Reply:
Good catch.  And the reformatting to proper JavaDoc is nice as well.